ruby/rental.rb

```diff
@@ -0,0 +1,13 @@
+class Rental
+  attr_accessor :date
```

Comment:
- [ ] For the same reason as in the request about the classroom and student classes, please add the person and book in the association with the rental class. 🙏